src/utils/azureUtils.ts

```diff
@@ -227,22 +223,40 @@ const supportedTypes: SupportedTypeMap = {
     'microsoft.web/serverfarms': { displayName: localize('serverFarms', 'App Service plans') },
     'microsoft.web/kubeenvironments': { displayName: localize('containerService', 'App Service Kubernetes Environment') },
     'microsoft.app/managedenvironments': { displayName: localize('containerAppsEnv', 'Container Apps Environment') },
-    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') },
+    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') }
+}
+
+export function isFunctionApp(resource: GenericResource): boolean {
+    const { type, kind } = resource;
+    if (type?.toLowerCase() === 'microsoft.web/sites') {
+        if (kind?.toLowerCase().includes('functionapp') && !kind?.toLowerCase().includes('workflowapp')) {
+            return true;
+        }
+    }
+    return false;
 }
 
-export function isFunctionApp(type?: string, kind?: string): boolean {
+export function isLogicApp(resource: GenericResource): boolean {
+    const { type, kind } = resource;
     if (type?.toLowerCase() === 'microsoft.web/sites') {
-        if (kind?.toLowerCase().includes('functionapp')) {
+        if (kind?.toLowerCase().includes('functionapp') && kind?.toLowerCase().includes('workflowapp')) {
             return true;
         }
     }
     return false;
 }
 
+export function isAppServiceApp(resource: GenericResource): boolean {
+    return !isFunctionApp(resource) && !isLogicApp(resource);
```

Comment:
This should include a resource type check, or else it would return true for everything that is not a function or logic app (e.g., true for a storage account).

Something like:
```suggestion
    return resource.type.toLowerCase() === 'microsoft.web/sites' &&
        !isFunctionApp(resource) &&
        !isLogicApp(resource);
```

Also, doesn't need to be in this PR but it's increasingly clear we need to define our own resource type enumeration--which has separate entries for App Service, Functions, and Logic Apps--and have a single function in utils to normalize `GenericResource` -> that type. Then we can stop needing all these `toLowerCase()` and type/kind checks all over.